native/machine-coding/fitness-tracker/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Fitness Tracker</title>
+    <script src="../../helpers/header.js" type="module"></script>
+    <link rel="stylesheet" href="style.css">
+    <script src="index.js" type="module"></script>
+</head>
+<body class="light-mode">
+    <div class="container light-mode">
+        <h1 class="light-mode">Fitness Tracker</h1>
+        <button id="dark-mode-toggle" onclick="toggleDarkMode()">Mode</button>
+
+        <!-- Fitness Tracker Content -->
+        <div id="workout-form">
+            <h2>Add a Workout</h2>
+            <label for="workout">Workout Type: </label>
+            <input type="text" id="workout">
+            <label for="duration">Duration (minutes): </label>
+            <input type="number" id="duration">
+            <button onclick="addWorkout()">Add</button>

```

Comment:
Looks like none of the js functions are working. You may try using `addEventListener` in the script instead.